packages/perseus/package.json

```diff
@@ -64,9 +66,12 @@
         "prop-types": "^15.6.1",
         "react": "16.14.0",
         "react-dom": "16.14.0",
+        "react-popper": "^2.0.0",
```

Comment:
This is what we use to power Tooltip and Popover, so not a new dependency for KA.
WB exposes a thin wrapper `TooltipPopper`, but there is a ticket to limit it to only Tooltip and Popover types, so it would break for us within a few versions.